common/thorhelper/thorsoapcall.cpp

```diff
@@ -2082,10 +2082,11 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
         StringBuffer dbgheader, contentEncoding;
         bool chunked = false;
         size32_t read = 0;
+        bool srtn;
```

Comment:
minor: not sure what "srtn" is acronym for, maybe isClosed/socketClosed would be clearer?

Reply:
I'm old - its just short for status return :-)  I can change it to sockClosed.